check_mk_telegram-notify.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# Push Notification (using Telegram)
+# Push Notification with Emojis (using Telegram)
```

Comment:
The comment in line 2 should "not" be changed, because Check_MK uses it for GUI configuration, so existing installations would have an impact.

# Push Notification (using Telegram)